src/Symfony/Component/HttpKernel/Controller/ControllerResolverInterface.php

```diff
@@ -17,8 +17,6 @@
  * A ControllerResolverInterface implementation knows how to determine the
  * controller to execute based on a Request object.
  *
- * It can also determine the arguments to pass to the Controller.
```

Comment:
to be applied on 4.0?

Reply:
I guess also for 3.4 but no priority for me